var/spack/repos/builtin/packages/aocc/package.py

```diff
@@ -56,7 +61,6 @@ class Aocc(Package):
     depends_on("zlib-api")
     depends_on("ncurses")
     depends_on("libtool")
-    depends_on("texinfo")
```

Comment:
Why was this removed for every version?

Reply:
This was a long-standing issue.  Texinfo is not actually required for any version currently in the recipe.